unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneLifecycleHandler.cs

```diff
@@ -53,6 +53,21 @@ public SceneLifecycleHandler(ParcelScene ownerScene)
             sceneResourcesLoadTracker = new SceneResourcesLoadTracker();
             sceneResourcesLoadTracker.Track(owner.componentsManagerLegacy, Environment.i.world.state);
             sceneResourcesLoadTracker.OnResourcesStatusUpdate += OnResourcesStatusUpdated;
+   
+            // This is done while the two ECS are living together, if we detect that a component from the new ECS has incremented a 
+            // resource for the scene, we changed the track since that means that this scene is from the new ECS.
+            // This should disappear when the old ecs is removed from the project. This should be the default track 
+            DataStore.i.ecs7.pendingSceneResources.OnRefCountUpdated += ChangeTrackingSystem;
```

Comment:
This looks like a really weird way of solving an issue.

Cannot we detect the version of the SDK before adding it to the tracker? Shouldnt the scenes bring a sdk version on its metadata?

Reply:
Right now we don't identify scenes by versions or any means. Since the ECS7 will be the default in the future, I talked with pato and this is an easy way to implement it until we can remove all the code.

However, I will change this event for this one `DataStore.i.ecs7.componentsManagers.OnAdded`